app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -76,10 +93,25 @@ class PatientSearchResultItemView(
     }
   }
 
-  private fun renderPatientPhoneNumber(phoneNumber: String?) {
-    phoneNumberContainer.visibleOrGone(phoneNumber.isNullOrBlank().not())
-    if (phoneNumber != null) {
-      phoneNumberTextView.text = phoneNumber
+  private fun renderPatientPhoneNumber(searchQuery: String?, patientSearchResult: PatientSearchResultViewModel) {
+    if (patientSearchResult.phoneNumber == null) {
+      phoneNumberContainer.visibility = View.GONE
+      return
+    }
+
+    phoneNumberContainer.visibility = View.VISIBLE
+    val canHighlightNumber = !searchQuery.isNullOrBlank() && patientSearchResult.phoneNumber.toString().contains(searchQuery)
```

Comment:
We can move this inside `getPatientPhoneNumber` since that's the only place it's being used and we are already passing `PatientSearchResultViewModel` and search query there.